### PR TITLE
frontend: Update OperatorHub description text

### PR DESCRIPTION
**Pull request by rebeccaalpert**
Added text from Jira including link to IBM Marketplace. Please note that @alimobrem says the Open Marketplace link isn't currently working; let's hold off on merging until the link works.

Screen Shot 2019-12-20 at 2 16 03 PM

Fixes https://issues.redhat.com/browse/CONSOLE-1971.

Heads up @openshift/team-ux-review.

### CODE REVIEW

**Comment by benjaminapetersen** [comment]
/approve

Sounds good to hold for now.

**Comment by beanh66** [comment]
@rebeccaalpert Can you add the external link icon to the right of the "Open Marketplace" link? This helps demonstrate to users that this link opens a new window for the new application vs the Developer Catalog link will navigate within the same application.

**Comment by rebeccaalpert** [comment]
All set @beanh66:

Screen Shot 2020-01-03 at 3 33 15 PM

**Comment by beanh66** [comment]
Thanks! @rebeccaalpert Can we make sure it is changed to "Red Hat Marketplace" rather than Open Marketplace?

**Comment by rebeccaalpert** [comment]
How's this @beanh66?

Screen Shot 2020-01-20 at 10 15 25 AM

**Comment by rebeccaalpert** [comment]
Updated per Slack changes:
Screen Shot 2020-01-20 at 11 50 29 AM

File: frontend/public/components/utils/webhooks.tsx [hunk 1]
```diff
@@ -90,7 +90,7 @@ export const WebhookTriggers: React.FC<WebhookTriggersProps> = (props) => {
       setWebhookSecrets(_.compact(secrets));
       setLoaded(true);
     });
-  }, [isLoaded, canGetSecret]);
+  }, [isLoaded, canGetSecret, secretNames, namespace]);
```
Comment (spadgett) [inline frontend/public/components/utils/webhooks.tsx]: This change isn't right. We should remove from this PR. It's already fixed under #3929

Comment (rebeccaalpert) [reply frontend/public/components/utils/webhooks.tsx]: The linter added this one. I'll rebase and remove it, but I was getting test failures without it.

**Comment by rebeccaalpert** [comment]
/hold cancel

Per @itsptk, this is safe to merge at this point.

**Comment by itsptk** [comment]
> /hold cancel
> 
> Per @itsptk, this is safe to merge at this point.

The decided URL 'https://marketplace.redhat.com/en-us?utm_source=openshift_console' currently does not resolve to the marketplace site, but the marketplace team is aware of the 4.4 timeline via email with @robszumski and it is expected to be active by mid-march.

**Comment by rebeccaalpert** [comment]
@itsptk Thanks for clarifying.

**Comment by robszumski** [comment]
once logged in, I can confirm that this URL is valid

File: frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-page.tsx [hunk 2]
```diff
@@ -129,10 +129,14 @@ export const OperatorHubPage = withFallback(
...
-            appear in the <a href="/catalog">Developer Catalog</a>, providing a self-service
-            experience.
+            Hat. You can purchase commercial software through{' '}
+            <ExternalLink
+              href="https://marketplace.redhat.com/en-us?utm_source=openshift_console"
+              text="Red Hat Marketplace"
+            />
+            . You can install Operators on your clusters to provide optional add-ons and shared
+            services to your developers. After installation, the Operator capabilities will appear
+            in the <a href="/catalog">Developer Catalog</a> providing a self-service experience.
```
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-page.tsx]: This should use `Link`, not an `a` element

Comment (rebeccaalpert) [reply frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-page.tsx]: Pushed change in amended commit.

**Comment by spadgett** [comment]
/hold
for merge queue fix #4065

**Pull request merged**